less/_components/quicksilver/featured-jumbotron.less

```diff
@@ -92,3 +92,34 @@
   }
 }
 
+.featured-jumbotron-alternate {
```

Comment:
Why not use .featured-jumbotron.alternate? Then we can easily inherit the base and just tweak what we need to rather than restate everything.

Reply:
In the html i'm using it like that:
.featured-jumbotron.featured-jumbotron-alternate

But I do like your solution and it would shorten the alternate class name.